.gitignore

```diff
@@ -5,3 +5,4 @@
 /*.iml
 /flix.jar
 target
+.ensime*
```

Comment:
Emacs uses [Ensime](http://ensime.org/) to do IDE like things for Scala. It creates a .ensime dir, so I've added it to the .gitignore.